src/mixins/embed.js

```diff
@@ -105,13 +108,43 @@ export default {
     });
     const metadata = data && data.data.records[0].data.contentMetadata;
     const stakeholders = data && data.data.records[0].data.stakeholders;
+
+    const stakeholdersFirstId = (stakeholders[0] && stakeholders[0].entity && stakeholders[0].entity['@id']) || '';
+    let inputAddress = stakeholdersFirstId;
+    const addressLengthWithoutPrefixAnd1 = 38;
+    if (stakeholdersFirstId.startsWith('did:like:')) {
+      inputAddress = `like1${stakeholdersFirstId.slice(stakeholdersFirstId.length - addressLengthWithoutPrefixAnd1)}`;
+    } else if (inputAddress.startsWith('did:cosmos:')) {
+      inputAddress = `cosmos1${stakeholdersFirstId.slice(stakeholdersFirstId.length - addressLengthWithoutPrefixAnd1)}`;
+    }
+    let stakeholdersValidlikeWallet;
+    if (isValidAddress(inputAddress)) {
+      stakeholdersValidlikeWallet = changeAddressPrefix(inputAddress, 'like');
+    }
+    let address = '';
+    let stakeholdersName;
+    if (stakeholdersValidlikeWallet
+      || (stakeholders && stakeholders[0] && stakeholders[0].entity.name)) {
+      address = stakeholdersValidlikeWallet;
+      stakeholdersName = (stakeholders && stakeholders[0] && stakeholders[0].entity.name);
+    } else {
+      address = data && data.data && data.data.owner;
+    }
+    const likerData = await apiGetLikerDataByAddress(address)
+      .catch(() => {});
+    const displayName = (likerData && likerData.data && likerData.data.displayName)
+    || stakeholdersName
```

Comment:
Please add a comment `stakeholdersName` is only set if stakeholder wallet is used